src/Cpanel/PluginActions.php

```diff
@@ -13,6 +13,36 @@ class PluginActions extends AbstractPluginActions
     protected $dataStore;
     protected $logger;
     protected $request;
+    protected $userConfig;
+    protected $composer;
+
+    public static $CONFIG = array(
+        'debug' => false,
+        'featureManagerIsFullZoneProvisioningEnabled' => true,
+        'isDNSPageEnabled' => true,
+        'useHostAPILogin' => true,
+        'homePageCards' => array(
```

Comment:
FYI you could just do:
```
array('AlwaysOnlineCard','IPV6Card',...,'PurgeCacheCard');
```
Don't need to change but in the future lets avoid hard coding indexes.